src/utils/components/ErrorAlert.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC } from 'react';
+import { Link } from 'react-router-dom-v5-compat';
+
+import { Alert, AlertVariant, Stack, StackItem } from '@patternfly/react-core';
+import { useNetworkingTranslation } from '@utils/hooks/useNetworkingTranslation';
+import { isEmpty } from '@utils/utils';
+
+type ErrorAlertProps = {
+  error: any | Error;
+};
+
+const ErrorAlert: FC<ErrorAlertProps> = ({ error }) => {
+  const { t } = useNetworkingTranslation();
+  if (isEmpty(error)) return null;
+
+  return (
+    <Alert isInline title={t('An error occurred')} variant={AlertVariant.danger}>
+      <Stack hasGutter>
+        <StackItem>{error.message}</StackItem>
```

Comment:
If we're not sure of `error`'s type we should probably use an optional chaining operator here.